L/libjulia/common.jl

```diff
@@ -286,16 +285,17 @@ function build_julia(ARGS, version::VersionNumber)
         push!(dependencies, Dependency("LibOSXUnwind_jll", compat="0.0.5"))
     elseif version < v"1.7"
         push!(dependencies, Dependency("LibOSXUnwind_jll", compat="0.0.6"))
+        push!(dependencies, Dependency("LibUV_jll"))
```

Comment:
Doesn't this rather belong in your other PR? It seem irrelevant for building 1.5?

Reply:
This is needed here, because `LibUV` was in the common dependencies but adding it for 1.5.4 makes the build fail on linux i686.